multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -134,6 +142,9 @@ func (c *StaleResCleanupController) cleanupStaleResourcesOnMember() error {
 	if err := c.cleanupClusterInfoResourceExport(commonArea, resExpList); err != nil {
 		return err
 	}
+	if err := c.cleanLabelIdentityResourceExport(commonArea, resExpList); err != nil {
```

Comment:
Do we need to clean up ResourceImports too? In both leader and member clusters? @luolanzone